dev/scalafmt

```diff
@@ -17,7 +17,4 @@
 # limitations under the License.
 #
 
-# by default, format only files that differ from git master
-params="${@:---diff}"
-
-./build/mvn mvn-scalafmt_2.12:format -Dscalafmt.skip=false -Dscalafmt.parameters="$params"
+./build/mvn mvn-scalafmt_2.12:format -Dscalafmt.skip=false
```

Comment:
Even if we figure out what's up with parameters, this would need to be either scalafmt_2.12 or scalafmt_2.13 accordingly

Reply:
Yeah, right, noticed that one too. I will instead change this to put the older version of the plugin under the scala-2.12 profile, to get it out of the way of the 2.13 build, for now. If the plugin further updates, we can use this approach here.